homeassistant/components/philips_js/light.py

```diff
@@ -231,21 +249,24 @@ def _update_from_coordinator(self):
         if (cache_keys := _get_cache_keys(self._tv)) != self._cache_keys:
             self._cache_keys = cache_keys
             self._attr_effect_list = self._calculate_effect_list()
-            self._attr_effect = self._calculate_effect()
+            self._attr_effect = str(self._calculate_effect())
 
         if current and current["isExpert"]:
             if settings := _get_settings(current):
                 color = settings["color"]
 
-        mode, _, _ = _parse_effect(self._attr_effect)
+        effect = AmbilightEffect.from_str(self._attr_effect)
+        LOGGER.debug("Current effect: %s, effect")
```

Comment:
This can't remain. It will be logged every update interval.

Reply:
I will remove this if you wish. However it's only debug level logging (needs to be specifically turned on for the specific component to do any I/O). And there is currently no logging at all in the light component. When users have issues I think it would be nice to have some strategic places to put debug logs, to help with resolving issues and takes the guess work out. Especially with all these different TV models and various behaviour of the API.
But it's up to you of course..

Reply:
btw I removed the logs for now.